spec/features/support/providers/courses/editing_a_course_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Edit provider course cide" do
+  let(:user) { create(:user, :admin) }
+
+  before do
+    given_i_am_authenticated(user: user)
```

Comment:
Can we replace this with a single step that creates and authenticates the user? Would like to clear up some of the `let` and `before` blocks in these feature specs over time.

```suggestion
    given_i_am_authenticated_as_an_admin_user
```